src/NUnitTestAdapter/AdapterSettings.cs

```diff
@@ -178,6 +199,18 @@ public void Load(string settingsXml)
                 DomainUsage = "None";
                 SynchronousEvents = true;
             }
+
+            // If DisableAppDomain settings is passed from the testplatform, set the DomainUsage to None.
+            if(DisableAppDomain)
+            {
+                DomainUsage = "None";
+            }
+
+            // If DisableParallelization setting is passed from the testplatform, set the NumberOfTestWorkers to 0
+            if(DisableParallelization)
+            {
+                NumberOfTestWorkers = 0;
+            }
```

Comment:
Should we account for the possibility that the user has used the NumberOfTestWorkers setting directly? Cases are:

DisableParalllelization | NumberOfTestWorkers | Action
----------------------------|-------------------------------|----------
true      | 0        | Disable 
true      | > 0     | Conflict - error message?
false     | 0       | Disable
false     | > 0    | Conflict - see note

NOTE: Currently we don't distinguish between the setting being missing and being set to false. Since that's the case, I would be inclined to allow parallel execution in the fourth row. If we were to distinguish the two, then I'd give a message if `DisableParallelization` were explicitly set to false while `NumberOfTestWorkers` was positive.

Reply:
Will take action only if `DisableParallelization` is `true`. If not will not take any action.

DisableParalllelization | NumberOfTestWorkers | Action
-- | -- | --
true | 0 | Disable
true | > 0 | Conflict - error message? [Is Warning Better than error message?]
false | 0 | Disable [Will not take any action, since `NumberOfTestWorkers` is already set]
false | > 0 | Conflict - see note [Will not take any action, so it will be parallel]

Reply:
If DisableParallelization is false, and NumberofTestWorkers > 0 then we have parallel.  There is no conflict there. 
(@Charlie: Did you switch the values there for DisableParallelization in the Note?)

For the 2nd case, DP = false and NOTW>0, I think it makes sense to make DP take precedence when true, and disable Parallel.  A message can be ok, but could also be set to display only for Verbosity>0.  All these messages are annoying in build outputs.

Reply:
You're right Terje, I switched two cases. I'm moving this discussion to the general comments because it gets hidden as a line comment when changes are made.